tool/Makefile

```diff
@@ -12,29 +12,41 @@ OBJS = *.o
 
 CFLAGS += -pthread -I ../include
 
+ifdef DEBUG
+CFLAGS += -g -O0
+endif
+
 LDFLAGS =
 
 LIBS = 
 
 PREFIX = /usr/local
 
+DIR := ${CURDIR}
+LIBDIR := ""
+ifndef DESTDIR
+	LIBDIR=$(DIR)/../lib/
+else
+	LIBDIR=$(DESTDIR)/lib
+endif
+
 tlspool_LIB = ../lib/libtlspool.so
-tlspool_LDFLAGS = -L ../lib -ltlspool
+tlspool_LDFLAGS = -L ../lib -ltlspool -Wl,-rpath,$(LIBDIR)
```

Comment:
I told you in the issue that RPATH is off-limits, it is a part of the design of NixOS but in any other system it is a hack and a cruel trick to play on unsuspecting admins.